liquibase-integration-tests/src/test/java/liquibase/statementexecute/AbstractExecuteTest.java

```diff
@@ -119,22 +109,22 @@ private void test(String[] expectedSql, Class<? extends Database>[] includeDatab
         }
 
         resetAvailableDatabases();
-        for (Database availableDatabase : DatabaseTestContext.getInstance().getAvailableDatabases()) {
-            Statement statement = ((JdbcConnection) availableDatabase.getConnection()).getUnderlyingConnection().createStatement();
-            if (shouldTestDatabase(availableDatabase, includeDatabases, excludeDatabases)) {
-                String sqlToRun = SqlGeneratorFactory.getInstance().generateSql(statementUnderTest, availableDatabase)[0].toSql();
-                try {
-                    for (SqlListener listener : Scope.getCurrentScope().getListeners(SqlListener.class)) {
-                        listener.writeSqlWillRun(sqlToRun);
-                    }
-                    statement.execute(sqlToRun);
-                } catch (Exception e) {
-                    System.out.println("Failed to execute against " + availableDatabase.getShortName() + ": " + sqlToRun);
-                    throw e;
-
-                }
-            }
-        }
+//        for (Database availableDatabase : DatabaseTestContext.getInstance().getAvailableDatabases()) {
```

Comment:
Confirming that you want all of the code in this file to be commented out, and not removed.

Reply:
Thanks, this was some code I needed to leave and had just meant to temporarily comment until I could refactor it correctly.

I'll get it working again